app/controllers/api/v1/regions_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Api::V1::RegionsController < ApplicationController
+    def index
+        render json: Region.all
```

Comment:
This controller should also be accompanied with some controller testing to ensure a `get :index` request is returning the correct json object

Reply:
Added rspec testing